flake.nix

```diff
@@ -28,14 +28,13 @@
             src = ./.;
             doCheck = true;
 
-            nativeBuildInputs = [ pkgs.makeWrapper ];
```

Comment:
Out of curiosity, why can you get away with not using `makeWrapper` anymore?

Reply:
This was legacy from before the grand nixtract Rust rewrite. At that point we were dependent on `nixtract` being available in `$PATH`, whereas after the grand RIIR, we just use the Rust crate.